sktime/forecasting/base/adapters/_pytorchforecasting.py

```diff
@@ -16,6 +18,15 @@
 
 from sktime.forecasting.base import ForecastingHorizon, _BaseGlobalForecaster
 
+try:
```

Comment:
the way I would solve it it:

* call the parameter `verbose` instead of `mute`, that is more common
* if `verbose=False`, set the `python_dependencies` tag to the higher `skbase` version
* always use the `skbase` classes

Reply:
actually, nevermind - let's have this in `sktime` but refactor later, since not all classes are available in `skbase`.